packages/localize/examples/runtime-ts/package.json

```diff
@@ -6,14 +6,45 @@
   "type": "module",
   "scripts": {
     "build": "wireit",
+    "build:localize": "wireit",
+    "build:ts": "wireit",
     "serve": "web-dev-server --node-resolve"
   },
   "wireit": {
     "build": {
-      "command": "rm -rf lib/* && lit-localize build && tsc",
+      "dependencies": [
+        "build:localize",
+        "build:ts"
+      ]
+    },
+    "build:localize": {
+      "command": "lit-localize build",
       "dependencies": [
         "../../../localize:build",
         "../../../localize-tools:build"
+      ],
+      "files": [
+        "lit-localize.json",
+        "tsconfig.json",
+        "src/**",
+        "xliff/**",
+        "!src/generated/**"
+      ],
+      "output": [
+        "src/generated/**"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc",
+      "dependencies": [
+        "build:localize",
```

Comment:
Oooh neat! I was about to comment for `build`'s dependency if the dependencies there need to be serial but this takes care of that.